cmd/prysm_compatibility_checker/main.go

```diff
@@ -80,6 +83,37 @@ func verify() *cobra.Command {
 	}
 }
 
+func verifyBatch() *cobra.Command {
+	return &cobra.Command{
+		Use:   "verify_batch [exported_signatures] [pubkey] [dir]",
```

Comment:
at first glance, I didn't get what `exported_signatures` is about. there is also no mention of it in the description of the command. I suggest to make it more obvious how to use the command, e.g. expand Short of the command with `./dc4bc_cli export_signatures` mention